components/src/core/components/Input/TreeSelect/TreeSelectText.vue

```diff
@@ -0,0 +1,182 @@
+<template>
```

Comment:
Do we need this component. Can't we reuse what is already there for multi select and introduce a slot to it?